Samples/Net5ProjectionSample/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <BuildOutDir>$([MSBuild]::NormalizeDirectory('$(SolutionDir.Replace("$(SolutionFileName)\", ""))', '_build', '$(Platform)', '$(Configuration)'))</BuildOutDir>
+    <BuildOutDir>$([MSBuild]::NormalizeDirectory('$(SolutionDir)', '_build', '$(Platform)', '$(Configuration)'))</BuildOutDir>
```

Comment:
ok, but generally, we don't remove workarounds (especially if they carry no risk) - users could still try to open the sample in the version of VS with the break.